fiftyone/core/collections.py

```diff
@@ -5814,14 +5817,18 @@ def geo_near(
             # Sort the samples by their proximity to Times Square
             #
 
-            view = dataset.geo_near(TIMES_SQUARE)
+            view = dataset.geo_near(TIMES_SQUARE, create_index=True)
```

Comment:
could be nice to have a debug log that a spherical index is being created since it could cause system slowdowns during the build process